utilities/wannier2sparse/include/operator_utils.hpp

```diff
@@ -35,7 +35,8 @@ namespace oputil
 	string AVAIL_OPS[] = {	
 							"VX"  ,"VY"  ,"VZ","VXSX","VXSY","VXSZ",
 							"VYSX","VYSY","VYSZ","VZSX","VZSY","VZSZ",
-							"SX", "SY", "SZ","TX","TY", "TZ"	
+							"SX", "SY", "SZ", "PUSX", "PUSY", "PUSZ",
+							"PDSX", "PDSY", "PDSZ", "TX","TY", "TZ"	

```

Comment:
Pushed the torque operators back to keep the projection operators next to the spin ones. Decided on PxSy where _x_ can be U for **U**P and D for **D**own and _y_ the usual X, Y, Z spin directions. Due to the unfortunate coincidence of naming the projection operator for the up, y spin _PUSY_ , I'm open to changes in the naming scheme

Reply:
I think this naming convention is fine, sometimes is good to have a laugh.